django/db/backends/oracle/compiler.py

```diff
@@ -0,0 +1,55 @@
+from django.db.models.sql import compiler
+
+
+class SQLCompiler(compiler.SQLCompiler):
+    pass
+
+
+class SQLInsertCompiler(compiler.SQLInsertCompiler, SQLCompiler):
+    pass
+
+
+class SQLDeleteCompiler(compiler.SQLDeleteCompiler, SQLCompiler):
+    pass
+
+
+class SQLUpdateCompiler(compiler.SQLUpdateCompiler, SQLCompiler):
+    pass
+
+
+class SQLAggregateCompiler(compiler.SQLAggregateCompiler, SQLCompiler):
+    pass
```

Comment:
No need for subclassing here. Maybe we should define `django.db.models.sql.compiler.__all__ = ('SQLCompiler', ...)` here and simply change this one to 

```suggestion
from django.db.models.sql import compiler
from django.db.models.sql.compiler import *
```